python/PEDvPUDplot.py

```diff
@@ -2,104 +2,166 @@
 """
 Created on Sat Nov  7 12:59:47 2015
 
-@author: ymseah
+Make scatterplot of pairwise uncorrected vs evolutionary distances.
+
+Pairwise uncorrected distances are obtained from SDT 1.0 (filename_mat.txt), while pairwise maximum likelihood distances are obtained from Tree-Puzzle 5.2 (outdist.txt). Both files are read to extract sequence ID and distance values.
+
 """
 
 import numpy as np
 import matplotlib.pyplot as plt
-import itertools # for zipping, less memory consumption than zip
+import itertools
 
 filepath = raw_input("Enter path to SDT matrix file: ")
 filepath2 = raw_input("Enter path to TreePuzzle distance output file: ")
 
-# function to get sequence IDs from SDT matrix output file
 def getSDTid(filepath):
+    """
+    Reads SDT 1.0 matrix output (filename_mat.txt) and returns list of sequence IDs.
```

Comment:
Will provide sample file. I realized later that I should've just written this as one script rather than trying to make separate functions, since I need this for the specific purpose of working with these specific files.